core/activity/src/http_proxy.rs

```diff
@@ -74,8 +80,32 @@ async fn proxy_http_request(
             .call_streaming(msg)
     });
 
+    if let Some(value) = request.headers().get(header::ACCEPT) {
+        if value.eq(mime::TEXT_EVENT_STREAM.essence_str()) {
+            return Ok(Either::Left(stream_results(stream)?));
+        }
+    }
+    Ok(Either::Right(await_results(stream).await?))
+}
+
+fn stream_results(
+    stream: impl Stream<Item = Result<Bytes, Error>> + Unpin + 'static,
+) -> crate::Result<impl Responder> {
     Ok(HttpResponse::Ok()
         .keep_alive()
         .content_type(mime::TEXT_EVENT_STREAM.essence_str())
         .streaming(stream))
 }
+
+async fn await_results(
+    mut stream: impl Stream<Item = Result<Bytes, Error>> + Unpin,
+) -> crate::Result<impl Responder> {
+    let response = stream.next().await;
+
+    if let Some(Ok(bytes)) = response {
+        let response_body = String::from_utf8(bytes.to_vec())
```

Comment:
Why do you assume that response will be encoded as utf8?

Reply:
I still heaven't created a full-featured headers support. I guess I would need to parse the Content-Type from the response and see if it is any different than Utf-8. This will come later